parser/ParseBasicExpressions.hpp

```diff
@@ -392,6 +400,13 @@ class ParseFunctionCall : public ParseExpression {
   }
 
   /**
+   * @return Whether this function call contains the DISTINCT operator.
```

Comment:
Right. There are two scenarios for `DISTINCT`:

(1) Apply to `SELECT` items, e.g.
```
SELECT DISTINCT x, y + z
FROM a
```
This will be represented as an aggregation with no aggregate function but group by expressions `x`, `y + z`, which is not related to this PR.

(2) Apply inside aggregate function, e.g.
```
SELECT SUM(DISTINCT x)
FROM a
GROUP BY y;
```
This will be treated as the `DISTINCT` case of the sum aggregation. There will be methods inside `AggregationHandleSum` to handle this.